Autotracking Update

TheSirArchibald opened the pull request:
Added autotracking to your pack.

Two new variants were added one for items only and the other for map tracking.

TheSirArchibald committed:
updated to include nes and names

TheSirArchibald committed:
combine for autotracking

TheSirArchibald committed:
updated for autotracking

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

update to overall layout so autotracking can work

TheSirArchibald committed:
Delete locations/all_sublocations.json

TheSirArchibald committed:
Add files via upload

update to autotracker

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

fix to add tracking

TheSirArchibald committed:
Update autotracker.lua

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

TheSirArchibald committed:
Add files via upload

fix for flags

TheSirArchibald committed:
Add files via upload

fix file for flags in map auto

TheSirArchibald committed:
Add files via upload

take off alarm flute active

Ars-Ignis commented on manifest.json:
Changing the package_uid means users won't get notified for the update, which seems like a problem.  Instead this will show up as a completely new package.  I think we can just change the author, though. Also, since the author field is a user-facing string, replace the underscore with a space.

Ars-Ignis commented on locations/east_cave_locations.json:
I'm not wild about the split overworld boxes, tbh.  I think I'd prefer to duplicate these files, and only use the split boxes for the autotracker map variant.  I can do that in a separate submission, though.

Ars-Ignis reviewed (commented):
Overall, this looks good.  I'm going to try and get a run in with it some time this week before I approve and merge this, and I don't mind making the few changes mentioned in my other comments here.

TheSirArchibald committed:
Updated Split chest locations & original locations

TheSirArchibald committed:
Fix to split chest

updated to keep my autotracker layout seperate

TheSirArchibald committed:
Update manifest.json

updated so pack doesn't seperate

TheSirArchibald commented:
Updated per your request.  Should be good for you to test.  I might want to update the UI to put the Wall/Bosses/Misc on the Item Tab to reduce the amount of navigation required when using autotracking.

TheSirArchibald committed:
Update to Layout

remove of wall/boss tab and include everything into the main item tracker

TheSirArchibald commented:
Updated layout added.  I think this makes more sense for the autotracker.

TheSirArchibald committed:
moved total trades icon

moved total trades

TheSirArchibald committed:
Update autotracker.lua

forgot to add the ingame address back in

TheSirArchibald committed:
layout change

Final layout change

TheSirArchibald committed:
Update init.lua

Didn't need split file for this

TheSirArchibald committed:
mimic logic fix

updated maps for mimic flag

TheSirArchibald committed:
Eu fix to overworld maps

Eu fix to overworld maps

TheSirArchibald committed:
Start to implment the flag reading

Start to implment the flag reading

TheSirArchibald committed:
Update autotracker.lua

adding the flag tracking start of code

TheSirArchibald committed:
Delete scripts/flagread.lua

not needed

TheSirArchibald committed:
Flag Tracking Added

update per code's fix

TheSirArchibald committed:
Map Switch

Not able to do this is Emotracker but can in poptracker.  Note not working yet.

TheSirArchibald committed:
auto flag change

stops user toggle of wt wu

TheSirArchibald committed:
Update init.lua

fix to stop user input from wt wu

TheSirArchibald committed:
Update autotracker.lua

Updated for read flag to run only once.  This is to account for in game changes to flags for mystery settings

TheSirArchibald committed:
update for gomode flag

based on the 15 elements required. No updated to work with Story mode off yet.

TheSirArchibald committed:
Update input_layouts_unid.json

updated for gomode flag and opel tracking instead of vampire and bug

TheSirArchibald committed:
Delete scripts/MapSwitching directory

TheSirArchibald committed:
Update autotracker.lua

fix for boss tracking if you don't have kill sword
